README_zh.md

```diff
@@ -76,7 +76,7 @@ $ make docker-build
 ```
 
 ### 部署
-请参考 [docs/installer/install_zh.md](docs/installer/installation_zh.md)
+请参考 [docs/installation/installation_zh.md](docs/installation/installation_zh.md)
```

Comment:
I mean we can remove this `docs/installation/installation_zh.md` to `docs/docs_zh/installation.md`, and remove `docs/installer/install.md` to `docs/installation.md`. I think we'll only have one install doc so this install dir is not necessary?

Reply:
oh, get it, comments are addressed, PTAL